.github/chatmodes/PAW-01A Spec Agent.chatmode.md

```diff
@@ -23,12 +23,12 @@ Optional external/context knowledge (e.g., standards, benchmarks) is NOT auto‑
 
 ## Start / Initial Response
 Before responding, inspect the invocation context (prompt files, prior user turns, current branch) to infer starting inputs:
-- Check for `WorkflowContext.md` in chat context or on disk at `.paw/work/<feature-slug>/WorkflowContext.md`. If present, extract Target Branch, Work Title, Feature Slug, GitHub Issue, Remote (default to `origin` when omitted), Artifact Paths, and Additional Inputs before asking the user for them.
+- Check for `WorkflowContext.md` in chat context or on disk at `.paw/work/<feature-slug>/WorkflowContext.md`. If present, extract Target Branch, Work Title, Feature Slug, Issue URL (or GitHub Issue for backward compatibility), Remote (default to `origin` when omitted), Artifact Paths, and Additional Inputs before asking the user for them.
 - Issue link or brief: if a GitHub link is supplied, treat it as the issue; otherwise use any provided description. If neither exists, ask the user what they want to work on.
 - Target branch: if the user specifies one, use it; otherwise inspect the current branch. If it is not `main` (or repo default), assume that branch is the target.
 - **Work Title and Feature Slug Generation**: When creating WorkflowContext.md, generate these according to the following logic:
   1. **Both missing (no Work Title or Feature Slug):**
-     - Generate Work Title from GitHub Issue title or feature brief (existing logic)
+     - Generate Work Title from issue title or feature brief (existing logic)
```

Comment:
I'm not sure what "existing logic" means here - I know it's not part of the changes, but looks like AI Cruft that slipped by. Can you validate it's not needed and delete?